packages/react-core/src/components/ApplicationLauncher/examples/ApplicationLauncher.md

```diff
@@ -6,14 +6,15 @@ propComponents: ['ApplicationLauncher', 'ApplicationLauncherItem']
 typescript: true
 ---
 
-Note: Application launcher is built on Dropdown, for extended API go to [`Dropdown`](/components/dropdown/) documentation.
-To add a tooltip, use the `tooltip` prop and optionally add more tooltip props by using `tooltipProps`. For more tooltip information go to [`Tooltip`](/components/tooltip/).
-
 import { ApplicationLauncher, ApplicationLauncherContent, ApplicationLauncherIcon, ApplicationLauncherText, ApplicationLauncherItem, ApplicationLauncherGroup, ApplicationLauncherSeparator, Text } from '@patternfly/react-core';
 import { HelpIcon, StarIcon } from '@patternfly/react-icons';
 import { Link } from '@reach/router';
 import pfIcon from './pf-logo-small.svg';
 
+Note: Application launcher is built on Dropdown, for extended API go to [Dropdown](../../components/dropdown/) documentation.
```

Comment:
I think you've gone 1 level too high here and for Tooltip below
![Screen Shot 2020-04-01 at 9 31 25 AM](https://user-images.githubusercontent.com/8651509/78142980-9ad72d00-73fb-11ea-90c8-9558efeabe0f.png)

Reply:
What matters is that it works as as relative path in the docs... I just pushed an update that I think will work.

Reply:
Pushed another update to use the absolute path. Hopefully it builds in patternfly-org with the path prefix okay!